src/StateBridge.sol

```diff
@@ -160,15 +185,21 @@ contract StateBridge is FxBaseRootTunnel, Ownable {
     function _sendRootToPolygon(uint256 root, uint128 timestamp) internal {
         bytes memory message;
 
-        // This encoding is specified as the encoding of the `bytes` received by
-        // `_processMessageFromRoot` in the Polygon state bridge. Specifically, it requires an ABI-
-        // encoded tuple of `(uint256 newRoot, uint128 supersedeTimestamp)`.
-        message = abi.encode(root, timestamp);
+        message = abi.encodeCall(IPolygonWorldID.receiveRoot, (root, timestamp));
```

Comment:
This can just use `PolygonWorldID` directly. `IPolygonWorldID` doesn't need to exist.